contributors/join-us/recruiting.md

```diff
@@ -19,57 +19,127 @@ If you're considering a role at Mattermost, here are the **Top 10 Reasons to wor
 9. **Learn, Master, Teach:** Building new skills, becoming experts, and training others is core to our culture.
 10. **Friends around the world:** Build relationships with staff and contributors in dozens of countries.
 
-## Interview Stages
+In addition to this page, we also encourage anyone interested in interviewing at Mattermost to read [About Mattermost](https://handbook.mattermost.com/company/about-mattermost). 
+
+## Working and Meeting Across the Globe
+
+### Work Anywhere
+
+We’re headquartered in Palo Alto, California and, like many open source companies, a large portion of Mattermost, Inc. works across the globe. We run as a distributed, high performance and high trust team using a few principles:
+
+#### 1. Work Anywhere with Good Internet
+
+Working from anywhere means more time with loved ones, no commute, and controlling everything about your workspace.
+
+The one requirement is having the ability to occassionally [join video calls using Zoom](https://support.zoom.us/hc/en-us/articles/201362023-System-Requirements-for-PC-Mac-and-Linux) between 8am and 1pm California time, which requires a good enough internet connection.
+
+Recommended download/upload speeds should be 8 Mbps/512 kbps for smooth group calling and screensharing, with a bare minimum of 4 Mbps/128 kbps. The [Ookla Speed Test](https://www.speedtest.net/) can be used to check your internet speed.
+
+There are exceptions to this requirement, for example: part-time roles that don’t need to take meetings, or roles providing support outside core meeting hours. Hiring Managers and Recruiters will indicate any time zone requirements for specific roles and/or teams.
+
+We are continually expanding where we can hire staff, updated lists can be found [here](https://handbook.mattermost.com/contributors/join-us) 
+
+#### 2. Be Explicit
```

Comment:
Do you mean being precise?

Reply:
I like precise. I had copied this portion from the core handbook page - https://docs.mattermost.com/process/working-at-mattermost.html#be-explicit but I think precise sets a better tone and accomplishes the same intent. Thanks @jasonblais!